packages/webapp/src/models/csv/index.ts

```diff
@@ -15,6 +15,7 @@ export interface ICsvTableHeader {
 	fieldName: string
 	use: boolean
 	hasSensitiveZeros: boolean
+	spreadWithDelimiter?: string
```

Comment:
I am wondering if we should make the spreadWithDelimiter nullable instead of optional and then instantiate the values to null to begin with. This would make it easier on the state management side of things especially for the cases where a user deselect a column from multi value option that was previously marked as multi value. Instead of having to iterate through objects, spread them and then delete the keys, we can just set them to null. Either way will work so I am fine if you want to leave the property as optional instead of nullable.

Reply:
This makes sense. Just pushed the new version